src/test/java/seedu/address/testutil/LeaveBuilder.java

```diff
@@ -87,11 +91,11 @@ public LeaveBuilder withDescription(String description) {
      * Sets the {@code status} of the {@code Leave} that we are building.
      */
     public LeaveBuilder withStatus(String status) {
```

Comment:
similar to above, can make withStatus take in a status enum? in case we change the naming

Reply:
same as about, the Status constructor is private